code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -200,100 +200,6 @@
 	sec_hud_set_security_status()
 	..()
 
-
-
-
-//Cycles through all clothing slots and tests them for destruction
-/mob/living/carbon/proc/shred_clothing(bomb,shock)
-	if(back)
-		back.shred(bomb,shock-20,src)
-	if(head)
-		head.shred(bomb,shock,src)
-	if(wear_mask)
-		wear_mask.shred(bomb,shock,src)

```

Comment:
I had to remove shred_clothing()  since ex_act()'s ..() already calls contents_explosion. So it was either this or have human/ex_act() not call ..() which would be inconsistent with every other ex_act(). We lose some accuracy in the process sadly, a suit with some bomb armor doesn't protect your jumpsuit from bomb effect anymore, unless you have bomb suit + bomb hood on you. This is a nerf.